# 👋 Hi, I'm Kuan-Hao Chao <img src="https://kuanhao-chao.github.io/Kuanhao-Chao/kuanhao_chao.png">


🎓 I'm currently a third-year Ph.D. Candidate in Computer Science at the Center for Computational Biology, Johns Hopkins University, working with <a target="_blank" href="https://scholar.google.com/citations?user=sUVeH-4AAAAJ&amp;hl=en" style="color:#4A4F53">Steven Salzberg</a> and <a target="_blank" href="https://scholar.google.com/citations?user=fKjqGyEAAAAJ&amp;hl=en" style="color:#4A4F53">Mihaela Pertea</a>. . My academic journey started in [Electrical Engineering](https://web.ee.ntu.edu.tw/eng/index.php) at [National Taiwan University (NTU)](https://www.ntu.edu.tw/english/index.html), shifting towards computer science in my final year at the [College of Engineering & Computer Science](https://cecs.anu.edu.au) at [Australian National University (ANU)](https://www.anu.edu.au) 🦘🐨


<br>

🧬 My research interest intersects deep learning with genomics and transcriptomics:

<ul><li>In <strong>transcriptional regulatory networks</strong>, my work uses sequence models to decode DNA patterns, aiming to uncover insights into how cis-regulatory DNA sequences and trans-regulators interact. I am developing a yeast large language model (LLM) to better understand the mechanisms of yeast gene expression regulation.</li><li>In <strong>splice site predictiong</strong>, I built a deep dilated residual convolutional neural network to decode the complexities of RNA splicing, alternative splicing, and the impact of genetic variants on cryptic splicing (<a href="https://www.biorxiv.org/content/10.1101/2023.07.27.550754v2">Learn more</a>).</li><li>In <strong>genome assembly</strong>, I assembled and annotated the first gapless Southern Chinese Han genome, <a href="https://www.ncbi.nlm.nih.gov/datasets/genome/GCA_024586135.1/">Han1</a>, using PacBio HiFi and Oxford Nanopore long reads, with T2T-CHM13 as a guide (<a href="https://doi.org/10.1093/g3journal/jkac321">Learn more</a>).</li><li>In <strong>pangenome indexing</strong>, I applied new renaming heuristics and an SMT solver to make the Wheeler graph recognition problem computationally feasible (<a href="https://doi.org/10.1016/j.isci.2023.107402">Learn more</a>).</li><li>In <strong>genome annotation</strong>, I used graph-based methods to stitch together fragmented DNA and protein alignments, thereby assembling them into more accurate annotations. (<a href="https://doi.org/10.1101/2024.05.16.593026">Learn more</a>). <!-- - My **transcriptome assembly** work focuses on modeling RNA-Seq data using directed acyclic splice graphs, with ongoing research into graph neural networks to decode the complexities of RNA splicing. ([Learn more](https://www.biorxiv.org/content/10.1101/2023.07.27.550754v2)). --></li></ul>

<br>

💻 I am an advocate for open-source software, embracing the philosophy of <strong>“build what you need, use what you build”</strong>. I invite you to explore my <strong><a href="https://khchao.com/news/">NEWS</a></strong> page for the latest updates on my projects.

💬 Feel free to reach out to me for collaborations, discussions, or just to say hi! **[Coffee chat! ☕️](https://calendly.com/kuanhao-chao/30min)**

🔍 Discover more about my work on my **[personal website](https://kuanhao-chao.github.io)**.
